src/ConfigProvider.php

```diff
@@ -0,0 +1,365 @@
+            Helper\EscapeJs::class            => InvokableFactory::class,
+            Helper\EscapeCss::class           => InvokableFactory::class,
+            Helper\EscapeUrl::class           => InvokableFactory::class,
+            Helper\Gravatar::class            => InvokableFactory::class,
+            Helper\HtmlTag::class             => InvokableFactory::class,
+            Helper\HeadLink::class            => InvokableFactory::class,
+            Helper\HeadMeta::class            => InvokableFactory::class,
+            Helper\HeadScript::class          => InvokableFactory::class,
+            Helper\HeadStyle::class           => InvokableFactory::class,
+            Helper\HeadTitle::class           => InvokableFactory::class,
+            Helper\HtmlFlash::class           => InvokableFactory::class,
+            Helper\HtmlList::class            => InvokableFactory::class,
+            Helper\HtmlObject::class          => InvokableFactory::class,
+            Helper\HtmlPage::class            => InvokableFactory::class,
+            Helper\HtmlQuicktime::class       => InvokableFactory::class,
+            Helper\InlineScript::class        => InvokableFactory::class,
+            Helper\Json::class                => InvokableFactory::class,
+            Helper\Layout::class              => InvokableFactory::class,
+            Helper\PaginationControl::class   => InvokableFactory::class,
+            Helper\PartialLoop::class         => InvokableFactory::class,
+            Helper\Partial::class             => InvokableFactory::class,
+            Helper\Placeholder::class         => InvokableFactory::class,
+            Helper\RenderChildModel::class    => InvokableFactory::class,
+            Helper\RenderToPlaceholder::class => InvokableFactory::class,
+            Helper\ServerUrl::class           => InvokableFactory::class,
+            Helper\Url::class                 => InvokableFactory::class,
+            Helper\ViewModel::class           => InvokableFactory::class,
+        ];
+    }
+
+    /**
+     * Default helper aliases
+     *
+     * Most of these are present for legacy purposes, as v2 of the service
+     * manager normalized names when fetching services.
+     *
+     * @return array<string,string>|array<array-key, string>
+     */
+    public static function defaultViewHelperAliases(): array
+    {
+        return array_merge(
+            self::zendFrameworkHelperAliases(),
+            self::legacyNormalizedHelperAliases(),
+            self::standardHelperAliases()
+        );
+    }
+
+    /**
+     * These aliases provide compatibility with legacy Zend helpers
+     *
+     * @todo Remove this method in 3.0
+     * @return array<string,string>
+     */
+    private static function zendFrameworkHelperAliases(): array
+    {
+        return [
+            // @codingStandardsIgnoreStart
+            'Zend\View\Helper\Asset'               => Helper\Asset::class,
```

Comment:
I'd prefer to finally drop zend bridge functionality rather than adding these here one-by-one

Reply:
This is solved in 3.0, but the changes here would need to be addressed in 3.0 also. If you think it's OK to kill off all the Zend stuff in a minor, then I'm happy to incorporate that into this patch